<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowAccountInfo</class>
 <widget class="QWidget" name="ShowAccountInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>407</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ShowAccountInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="mainImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Variable Small Semibol</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../images.qrc">:/app/ui/images/no-profilepng.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showImages">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Variable Small Semibol</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Account images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadImage">
         <property name="font">
          <font>
           <family>Segoe UI Variable Small Semibol</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Load image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="AccountInfo">
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Account info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../images.qrc">
             <normaloff>:/app/ui/images/settings.ico</normaloff>:/app/ui/images/settings.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../images.qrc">
             <normaloff>:/app/ui/images/refresh.ico</normaloff>:/app/ui/images/refresh.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hideButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../images.qrc">
             <normaloff>:/app/ui/images/close.ico</normaloff>:/app/ui/images/close.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tagLine">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLine">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Phone number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phoneLine">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Variable Small Semibol</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLine">
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Creation date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="createDate">
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Segoe UI Variable Small Semibol</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="aboutLine">
     <property name="font">
      <font>
       <family>Segoe UI Variable Small Semibol</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
